src/CAN/can_mapping.c

```diff
@@ -20,8 +20,7 @@
  */
 #include "can_mapping.h"
 #include "byteswap.h"
-
-#include <stdio.h>
+#include "printk.h"
```

Comment:
yeah, not sure what happened there. Removed the printk.h since it's not being used. <shrug>